trussworks/react-uswds
# fix: Rollback husky to v4

# Note!!:
This is an **alternative** to https://github.com/trussworks/react-uswds/pull/925. We should merge one and close the other + delete the branch.

# Summary

Roll back to 4.3.8 per https://github.com/trussworks/react-uswds/commit/6bf67ba308eb949391a7fa5ca1ceae3cacf09426

Prevent dependabot updates to husky v5

closes: https://github.com/trussworks/react-uswds/issues/923

## How To Test

1. checkout this branch:
```
git fetch
git checkout -b bl-husky-rollback-to-v4-923
```

2. Perform a clean yarn install. Making a commit and verifying the hooks ran are also good tests. I did that as a part of pushing this commit up for this PR. Screenshot from danger on the pre-push for reference:
![image](https://user-images.githubusercontent.com/15805554/108567581-81bac180-72d6-11eb-8c06-60d9bce9fdcb.png)
